### DISCUSSION TITLE
Hi Folke, just wanna say you are really my top G with plugin

### DISCUSSION THREAD

**Discussion by radvil** [folke/flash.nvim#31] [General]
So much ez to customize, You are awesome, thank you for these much contribution man....

**Comment by folke** [comment]
Thank you!

**Comment by groovyghoul** [comment]
I second this! Everything that I use that you are a part of is top tier! It's made my life in neovim even more enjoyable! (even on my Windows setup)

**Comment by pogopaule** [comment]
I tried so many motion plugins. This one just works and feels so intuitive. Thank you so much!

**Comment by StephanRoemer** [comment]
I totally agree, Folke, you are on fire! Thank you for all your valuable contributions to the nvim community!
